dev-python/QtPy/QtPy-1.9.0-r3.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..8} )
+
+inherit distutils-r1 virtualx
+
+DESCRIPTION="Abstraction layer on top of PyQt5 and PySide2 and additional custom QWidgets"
+HOMEPAGE="https://github.com/spyder-ide/qtpy"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+
+IUSE="
+	3d bluetooth charts concurrent datavis dbus declarative designer
+	examples gles2-only gui help location multimedia network networkauth
+	opengl positioning printsupport qml quick script scripttools
+	scxml sensors serialport speech sql ssl svg test testlib webchannel
+	webengine webkit websockets widgets x11extras xml xmlpatterns
+"
+
+RDEPEND="
+	app-eselect/eselect-QtPy
+	|| (
+		dev-python/PyQt5[${PYTHON_USEDEP},designer?,gles2-only?,gui?,help?,location?,multimedia?,network?,opengl?,positioning?,printsupport?,sensors?,serialport?,sql?,svg?,testlib?,webchannel?,websockets?,widgets?,x11extras?,xml(+)?,xmlpatterns?]
+		dev-python/pyside2[${PYTHON_USEDEP},designer?,gles2-only?,gui?,help?,location?,multimedia?,network?,opengl(+)?,positioning?,printsupport?,sensors?,serialport(+)?,sql?,svg?,testlib?,webchannel?,websockets?,widgets?,x11extras?,xml?,xmlpatterns?]
```

Comment:
Since the USE flags here are optional, what is actually **required** for this program to work?

Reply:
Nothing is strictly required, it is an abstraction layer. The bare minimum would be only the qtcore module of PyQt5 or PySide2, which is always installed by PyQt5/PySide2 irrespective of the USE flag configuration.